.github/workflows/tool-tests.yml

```
@@ -1,39 +1,25 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Tool Tests
 
-# Controls when the action will run. 
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
 on:
-  # Triggers the workflow on push or pull request events but only for the develop branch
   push:
     branches: "**"
-  pull_request:
-    branches: "**"
 
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
-    # The type of runner that the job will run on
+  unit_test_execution:
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
-
-      # Runs a single command using the runners shell
-      - name: ls
-        run: ls -la
-      - name: GH-TOKEN Export
-        run: export GH_TOKEN=$(GH_TOKEN)
-      - name: install requirements
-        run: python -m pip install -r requirements.txt 
-      - name: Run a one-line script
-        run: python -m unittest ./python/tests/test_update_package_properties.py
-
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Install package dependencies
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+      - name: install python requirements
+        run: python -m pip install -r python/requirements.txt
+      - name: Execute Unit tests
+        run:  python -m pytest -q python/tests/test_update_package_properties.py
```

Consistent casing. If you do not like my changes, pick another style and stick to it.

```suggestion
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Install package dependencies
        run: sudo apt install libcurl4-openssl-dev libssl-dev
      - name: Install python requirements
        run: python -m pip install -r python/requirements.txt
      - name: Execute unit tests
        run:  python -m pytest -q python/tests/test_update_package_properties.py
```

---

Fixed